src/core/src/service_interfaces/TelemetryWriter.py

```diff
@@ -75,14 +77,14 @@ def write_package_info(self, package_name, package_ver, package_size, install_du
     # Composed payload
     def write_machine_config_info(self):
         # Machine info - sent only once at the start of the run
-        machine_info = {
+        TelemetryWriter.machine_info = {
```

Comment:
self.machine_info - here and further below

Reply:
fixed